pgcli_sublime.py

```diff
@@ -185,15 +187,8 @@ def run(self, edit):
 
         if not sql and len(sel) == 1:
             # Nothing highlighted - find the current query
-            sql = get_entire_view_text(self.view)
-            split_sql = sqlparse.split(sql)
-            curr_point = sel[0].a
-            cum_len = 0
-
-            for sql in split_sql:
-                cum_len += len(sql)
-                if curr_point <= cum_len:
-                    break
+            current_query = get_current_query(self.view)
+            sql = current_query[0]
```

Comment:
Similarly, this should be either `current_query, cursor_pos = get_current_query(self.view)` or, even better,  `current_query, _ = get_current_query(self.view)`, since cursor_pos isn't used